src/snowflake/connector/pandas_tools.py

```diff
@@ -36,7 +36,8 @@ def write_pandas(conn: 'SnowflakeConnection',
                  chunk_size: Optional[int] = None,
                  compression: str = 'gzip',
                  on_error: str = 'abort_statement',
-                 parallel: int = 4
+                 parallel: int = 4,
+                 quote_identifiers: bool = False
```

Comment:
By making this `False` by default is the part that makes this a breaking change. Unfortunately I have still not received approval for this, which is why I have been MIA from your PR @kyleflan